synoptic/src/synoptic/tests/SynopticTest.java

```diff
@@ -57,6 +58,30 @@ public void setUp() throws ParseException {
         super.setUp();
     }
 
+    /*
+     * Due to differences between command line and IDE usage, checks the path
+     * used and adjusts it accordingly.
+     */
+    public static String getTestPath(String path) throws Exception {
+
+        // original path passed in
+        File originalPath = new File(path);
+        // parent directory of the path passed in
+        File parentPath = new File("../" + path);
+        // removing parent directory (../) from path passed in
+        File parentRemovedPath = new File(path.substring(3));
```

Comment:
Ensure the path `startsWith("../")` or "..\\" first. Also, there are more robust ways to do these kinds of path manipulations in general, like using [Paths](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Paths.html) and [Path](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html), but I'm OK with these simple ways, since this is only used in our tests.